chain/actors/actors.go

```diff
@@ -37,10 +39,11 @@ func init() {
 		return c
 	}
 
-	AccountActorCodeCid = mustSum("account")
-	StorageMarketActorCodeCid = mustSum("smarket")
-	StorageMinerCodeCid = mustSum("sminer")
-	MultisigActorCodeCid = mustSum("multisig")
-	InitActorCodeCid = mustSum("init")
-	PaymentChannelActorCodeCid = mustSum("paych")
+	AccountActorCodeCid = mustSum("filecoin/1.0/AccountActor")
```

Comment:
Are these in the spec? Its really annoying to have them be so long, these objects get a lot of churn and doubling their size for no good reason bothers me.

@jbenet how about `fil/1/account`. Semver is worthless on chain, and theres no need to be extra verbose.